### PR TITLE
Making Frontend and Backend work together on localhost

### PR DESCRIPTION
**Pull request by offcast-dev**
fixing typos, naming, request calls, object keys, missing functionality in order to make front end and backent work together on localhost

### CODE REVIEW

File: cypress/integration/userCanReadMenuSortedByCategories.feature.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 describe("User can see menu", () => {
   beforeEach(() => {
-    cy.server();
+    /*cy.server();
```
Comment (DavDan1) [inline cypress/integration/userCanReadMenuSortedByCategories.feature.js:3]: Can you uncomment this?

Comment (DavDan1) [inline cypress/integration/userCanReadMenuSortedByCategories.feature.js]: Can you uncomment this?

File: src/components/ViewOrder.jsx [hunk 2]
```diff
@@ -5,31 +5,31 @@ import {Item, Container, Header, Button} from 'semantic-ui-react'
 class ViewOrder extends Component {
 
   state = {
-    menuData: [],
+    orderData: [],
```
Comment (DavDan1) [inline src/components/ViewOrder.jsx:8]: Why did you  change from menuData to orderData?

File: src/modules/orderHelper.js [hunk 3]
```diff
@@ -4,27 +4,28 @@ const authHeaders = JSON.parse(localStorage.getItem('userData'))
...
      {headers: authHeaders}) 
-  return response.data.body
+  return response.data
 }
 
 const getOrder = async (order_id) => {
-  const response = await axios.get(`/orders/${order_id}`, {headers: authHeaders})
-  return response.data.order.items;
+  const response = await axios.get(`http://localhost:3000/api/orders/${order_id}`, {headers: authHeaders})
+  return response.data.order.menu_items;
```
Comment (DavDan1) [inline src/modules/orderHelper.js:24]: why did it become menu_items?

**Review by DavDan1 — COMMENTED** [review]
Can you please resolve the issues?

File: cypress/integration/userCanReadMenuSortedByCategories.feature.js (hunk 1 above)
Comment (offcast-dev) [reply cypress/integration/userCanReadMenuSortedByCategories.feature.js]: thanks, missed it)

File: src/components/ViewOrder.jsx (hunk 2 above)
Comment (offcast-dev) [reply src/components/ViewOrder.jsx:8]: because menuData didn't reflect the contents of variable

File: src/modules/orderHelper.js (hunk 3 above)
Comment (offcast-dev) [reply src/modules/orderHelper.js:24]: because this is the correct key that backend is sending

**Review by DavDan1 — APPROVED** [review]

**Pull request merged**